src/core/packages/rendering/server-internal/src/rendering_service.tsx

```diff
@@ -243,6 +243,7 @@ export class RenderingService {
     const bootstrapScript = isAnonymousPage ? 'bootstrap-anonymous.js' : 'bootstrap.js';
     const metadata: RenderingMetadata = {
       strictCsp: http.csp.strict,
+      hardenPrototypes: http.csp.strict, // TODO[legrego]: use a dedicated configuration for this
```

Comment:
This approach makes sense to me.

Theoretically, we could use the feature flag service to drive this metadata. You wanted to control rollout separate from new code releases AND avoid overloading CSP config.

It would require passing the feature flag services' start contract to the rendering service. Right now it only has the setup contract passed to it.

Reply:
@jloleysens I'm of two minds, if I understand the feature flag service correctly -- on one hand, I want to give administrators a quick escape hatch to disable this if it breaks something that's not caught by tests or other monitoring. This use case feels best suited to yml config.

On the other hand, it would be great to progressively roll this out to users and monitor how it's going. This use case feels best suited to feature flags.

Can I have my cake and eat it too? 😄